project-canvas/src/components/backlog-view/IssuesWrapper/DraggableIssuesWrapper.tsx

```diff
@@ -20,8 +20,11 @@ export function DraggableIssuesWrapper({
             spacing="sm"
           >
             {issues.map((issue: Issue, index) => (
-              <IssueCard {...issue} key={issue.issueKey} index={index} />
+              <Box sx={{ position: "relative" }}>
```

Comment:
you can add the sx={{ position: "relative" }} to the Paper Component in the IssueCard.tsx like so 
<img width="475" alt="image" src="https://user-images.githubusercontent.com/54434958/223793540-5a367886-53ba-42c8-86e7-998848af731c.png">

Reply:
i tried but that gave me some performance issues